### DISCUSSION TITLE
Edit geojson file in lfs

### DISCUSSION THREAD

**Discussion by cadasa** [git-lfs/git-lfs#4662] [Q&A] [answered]
I have put a big geojson file in lfs but how can I get the url link to open the content of the original file (not the pointer as in this [link](https://cadasa.github.io/lsf/dcg.geojson))?

**Comment by bk2204** [comment, marked as answer]
Hey,

It looks like this is a GitHub Pages site, and last I checked, GItHub Pages didn't support Git LFS.  I'm pretty sure this is because in general a GitHub Pages site is supposed to be less than 1 GB in size, and most repositories using Git LFS are going to be larger than that.

If you have feedback about that, I'd contact GitHub Support.

**Reply by cadasa** [reply]
Hi,
Yes, this is a GitHub Page site. Do you mean that if it is not a page site then I can see the raw file?

thanks for your prompt reply!

**Reply by bk2204** [reply]
If you clone the repository and you have Git LFS installed and configured properly, then you'll be able to view the file.  However, when GitHub Pages clones the repository, it doesn't have Git LFS installed, so all you get is the pointer file that represents the large file, not the large file (the GeoJSON file) itself.  So this file will be visible in your repository, but it won't work properly on Pages.

**Reply by cadasa** [reply]
Thanks for your explanation! So what is the best way to upload it and get the url link of the Geojson file itself?

**Reply by bk2204** [reply]
If you want to host the GeoJSON file yourself, there are a variety of sites which allow you to do that,  You can host it in a public cloud bucket, on a cloud instance with a web server like nginx or Apache, or on a different hosting provider that supports Git LFS (e.g. Netlify) or has a non-Git related deployment mechanism.  The definition of "best" is going to differ here based on your needs and desires for hosting, and only you will know what works for you.

**Reply by cadasa** [reply]
Thanks again for your suggestion!
